packages/fluentui/react-builder/src/narration/NarrationComputer.tsx

```diff
@@ -209,85 +256,139 @@ export class NarrationComputer {
     return parent1;
   } // End getCommonAncestor
 
-  // Computes and returns the screen reader narration for the given element using the previous element and platform.
-  async getNarration(element: IAriaElement, prevElement: IAriaElement, platform: string): Promise<string> {
-    let definitionName = this.getDefinitionName(element, platform, 'stateRules');
-
-    // Retrieve the computed accessible node
-    const node = await (window as any).getComputedAccessibleNode(element);
-
-    // Compute and store all the narration parts
-    this.computeLandmarksAndGroups(element, prevElement, platform);
-    this.computeUsage(definitionName, element, platform);
-    this.computeDescription(definitionName, element, platform);
-    this.computeNameAndTitle(node, element);
-    this.computeValue(node);
-    this.computePosition();
-    this.computeTypeAndState(node, definitionName, element, platform);
-
-    definitionName = this.getDefinitionName(element, platform, 'readingOrder');
-    const computedNarration = this.composeNarrationFromParts(definitionName, platform);
-    return computedNarration;
-  } // End getNarration
-
-  // Returns the definition name based on the given element, platform and definition type.
-  getDefinitionName(element: IAriaElement, platform: string, definitionType: string): string {
-    // Determine the definitions source by the definition type
-    const definitionTypes: Record<string, any> = {
-      readingOrder: SRNC.readingOrder[platform],
-      stateRules: SRNC.stateRules[platform],
-    };
-    const definitions = definitionTypes[definitionType];
+  // Returns the platform inheritance list for the given platform
+  getPlatformInheritance(platform: string): string[] {
```

Comment:
Does it make sense to create a type for `platform` to constrain the strings that are allowed ?

Reply:
Well, not sure what benefit it will bring.

Reply:
easier maintenance/readability IMO

Reply:
Okay, changed.